src/main/java/org/broadinstitute/hellbender/tools/evoquer/EvoquerEngine.java

```diff
@@ -493,10 +500,10 @@ public int compare( GenericRecord o1, GenericRecord o2 ) {
     }
 
     private void processSampleRecordsForPosition(final long currentPosition, final String contig, final Iterable<GenericRecord> sampleRecordsAtPosition, final Set<String> columnNames) {
-        final List<VariantContext> unmergedCalls = new ArrayList<>();
+        List<VariantContext> unmergedCalls = new ArrayList<>();
```

Comment:
It might be better to leave these final and add a new unmergedCallsAfterSubsetting variable that gets filled in later.